src/App.js

```diff
@@ -110,6 +110,17 @@ function App() {
   return (
     <div className="App" ref={resumeRef}>
       <h1>Resume Builder</h1>
+      <div className="resumeSection">
+        <h2>Personal Information</h2>
+        <label for="name">Enter your full name: </label>
+        <input type="text" class="name"></input>
+        <br></br>
+        <label for="email">Enter your phone number: </label>
+        <input type="email" class="email"></input>
+        <br></br>
+        <label for="email">Enter your email address: </label>
+        <input type="tel" class="phoneNumber"></input>
+      </div>
```

Comment:
```suggestion
      <from className="resumeSection">
        <h2>Personal Information</h2>
        <label for="name">Enter your full name: </label>
        <input type="text" class="name"></input>
        <br></br>
        <label for="email">Enter your phone number: </label>
        <input type="email" class="email"></input>
        <br></br>
        <label for="email">Enter your email address: </label>
        <input type="tel" class="phoneNumber"></input>
        <button type="submit">Save</button>
      </from>
```

# Changes
1. wrap the inputs in a form to handle validation and submission
2. add submit button

# TODO
1. Style the inputs
2. write the js code to handle state and submission

> [!NOTE]
> This is a suggestion. I believe you can accept it directly from GitHub and the modifications will be applied to the code.